cmd/troubleshoot/cli/run.go

```diff
@@ -38,6 +42,38 @@ import (
 
 func runTroubleshoot(v *viper.Viper, args []string) error {
 	ctx := context.Background()
+	if len(args) == 1 && args[0] == "lint" {
```

Comment:
I assume you implemented this subcommand this way cause its a WIP PR, right?

I would have expected `lint` to be a cobra subcommand

Reply:
Yes, it is a WIP PR for showing what lint could be like. Sure, I will put it into `cmd/lint` as a cobra subcommand